src/pages/Home.jsx

```diff
@@ -59,6 +63,24 @@ export default function MiniDrawer() {
     setOpen(!open);
   };
 
+  async function getIssuesCount() {
+    try {
+      const issues = await axios.get('/issues');
+      setIssuses(issues.data.result.issues);
+      setIssusesCount(issues.data.result.issues.length);
+    } catch (error) {
+      console.log('error', error);
+    }
+  }
+
+  React.useEffect(() => {
+    const interval = setInterval(() => {
+      getIssuesCount();
+      console.log('This will run every 5 seconds!', issuses);
```

Comment:
I don't think we need it every 5 seconds. Let's maybe keep it to only once on the page load